polls/models.py

```diff
@@ -0,0 +1,91 @@
+from django.db import models
+from django.utils import timezone
+
+from authentication.models import User
+
+
+class Poll(models.Model):
+    # The question of the poll.
+    question = models.CharField(
+        'question',
```

Comment:
I struggled to find the meaning of this positional argument in the docs, https://docs.djangoproject.com/en/2.0/topics/db/models/#verbose-field-names

Maybe we can replace this line by 
```suggestion
        verbose_name='question',
```

So it's easier to understand